edu-services/gradebook-service/impl/src/java/org/sakaiproject/component/gradebook/GradebookServiceHibernateImpl.java

```diff
@@ -2815,6 +2815,17 @@ public void setGradebookPermissionService(GradebookPermissionService gradebookPe
 		this.gradebookPermissionService = gradebookPermissionService;
 	}
 
-	
+	@Override
+	public Object getGradebookWithGradeMappings(final Long id) {
```

Comment:
As above, I think it would be more useful to return the GradeMappings instead of the entire Gradebook.